.github/workflows/ci.yaml

```diff
@@ -12,12 +12,12 @@ on:
     branches: [ master ]
 
 env:
-  # Qt: 5.15.1
-  # libtorrent: RC_1_2 HEAD, 1.2.11
-  VCPKG_COMMIT: 133051b793486ef14e67e9d1f48c9cfe64dc127e
+  # Qt: 5.15.2
```

Comment:
maybe drop this? since we have no real control over it.

Reply:
>maybe drop this? 

Done.
Also I dropped comment about libtorrent version since we don't use libtorrent port from vcpkg repo anymore.